src/dialogs/ha-more-info-dialog.html

```diff
@@ -43,6 +43,16 @@
         :host {
           @apply(--ha-dialog-fullscreen);
         }
+        :host::before {
```

Comment:
Is this a fix that we'll need to apply to all places that use `ha-dialog-fullscreen`? If so, let's add a comment here and inside `ha-style`.

Reply:
Only if it’s contents can can change dynamically.
Currently can think of no other. Didn’t search all the code yet though...

Reply:
The more-info appears to be the only dialog that really needs this.
 The others are either non-fullscreen or don't have dynamic content 
Actually, hassio-markdown could be used with dynamic content... So I'll add this to that style too.
If do still have plans to gather and consolidate all these different dialogs (especially the styles) so we don't have our dialog-styles and media queries duplicated and sprinkled around in the codebase like we have now.

Reply:
config flow dialog also has dynamic content.

Reply:
yes, but that one doesn't have a fixed height (no toolbar), only a max-height.